openlibrary/templates/covers/add.html

```diff
@@ -54,6 +54,24 @@
     </div>
 </form>
 
+<form id="clipboardForm" class="ol-cover-form ol-cover-form--clipboard" method="post" enctype="multipart/form-data" action="$action">
+    <div class="label">
+        <label for="coverClipboard">$_("Or, paste an image from your clipboard")</label>
+    </div>
+
+    <div class="button-container">
+        <img id="image" />
+    </div>
```

Comment:
```suggestion
    <div class="image-container"></div>
```
It's kind of strange that we're starting with an invalid `img` tag here (`src` is required).

Instead, create the image element client-side, then insert it into the `.image-container` div.